Solutions/PingFederate/Analytic Rules/PingFederateForbiddenCountry.yaml

```diff
@@ -19,7 +19,7 @@ query: |
   let forbidden_geo = dynamic(['CH']);
   PingFederateEvent
   | where isnotempty(DstGeoCountry)
-  | where DstGeoCountry in~ (forbidden_geo)
+  | where tostring(DstGeoCountry) in~ (forbidden_geo)

```

Comment:
Getting KQL validation error : Errors:A value of type 'string' or 'dynamic' expected. 
hence updated this fix.